skglm/estimators.py

```diff
@@ -26,6 +29,120 @@
 )
 from skglm.utils import compiled_clone
 from skglm.solvers import cd_solver_path, bcd_solver_path
+from skglm.solvers.cd_solver import cd_solver
+
+
+def glm_fit(X, y, model, datafit, penalty):
```

Comment:
Should we make this private ? Otherwise we need a docstring

Reply:
I have added docstring for now. I can also be private function eventually

Reply:
I think it should be private. Do you envision a case where a user would need it?

Reply:
Not really, I changed it to a private function :)